### PR TITLE
Adding rel note about playback rate

### LINKED ISSUE #6684: [WebAPI] Setting playbackRate on streams
## Acceptance Criteria

- [x] The listed features are documented sufficiently on MDN
- [x] BCD is updated
- [x] Interactive example and data repos are updated if appropriate
- [x] The content has been reviewed as needed

### For folks helping with Firefox related documentation

- [x] Set bugs to `dev-doc-complete`
- [x] Add entry to Firefox release notes if feature is enabled in release
or
- [x] Add entry to Firefox experimental features page if feature is not yet enabled in release

## Features to document

- Setting playbackRate on a media element does not work with streams

## Related Gecko bugs

- [Setting playbackRate on a media element does not work when the media element is going to the MediaStreamGraph](https://bugzilla.mozilla.org/show_bug.cgi?id=1517199). Check the docs, might be updates to either audio or streams depending on if this was mentioned. Also a bcd check.

### PR DESCRIPTION
**Pull request by Rumyra**
### Description

Adding release note about playback rate missed from FF91

Fixes #6684

### CODE REVIEW

File: files/en-us/mozilla/firefox/releases/91/index.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ No changes
   and the {{domxref("Window.gamepadconnected_event", "gamepadconnected")}} and {{domxref("Window.gamepaddisconnected_event", "gamepaddisconnected")}} events will not fire.
   The default `allowlist` is `*`; this default will be updated to `self` in a future release, in order to match the specification. ([Firefox bug 1704005](https://bugzil.la/1704005)).
 - `Window.clientInformation` has been added as an alias for {{domxref("Window.navigator")}}, in order to match recent specification updates and improve compatibility with other major browsers ([Firefox bug 1717072](https://bugzil.la/1717072)).
+- Changing the speed of a media element ([<video>](/en-US/docs/Web/HTML/Element/video) or [<audio>](/en-US/docs/Web/HTML/Element/audio)) using the [`playbackRate` attribute](/en-US/docs/Web/API/HTMLMediaElement/playbackRate) now works when the media element is captured to a [`MediaStream`](/en-US/docs/Web/API/MediaStream) or via [`AudioContext.createMediaElementSource`](/en-US/docs/Web/API/AudioContext/createMediaElementSource)
```
Comment (Josh-Cena) [inline files/en-us/mozilla/firefox/releases/91/index.md:50]:
```suggestion
- Changing the speed of a media element ([`<video>`](/en-US/docs/Web/HTML/Element/video) or [`<audio>`](/en-US/docs/Web/HTML/Element/audio)) using the [`playbackRate`](/en-US/docs/Web/API/HTMLMediaElement/playbackRate) attribute now works when the media element is captured to a [`MediaStream`](/en-US/docs/Web/API/MediaStream) or via [`AudioContext.createMediaElementSource`](/en-US/docs/Web/API/AudioContext/createMediaElementSource)
```

File: files/en-us/mozilla/firefox/releases/91/index.md [hunk 2]
```diff
@@ -47,6 +47,7 @@ No changes
   and the {{domxref("Window.gamepadconnected_event", "gamepadconnected")}} and {{domxref("Window.gamepaddisconnected_event", "gamepaddisconnected")}} events will not fire.
   The default `allowlist` is `*`; this default will be updated to `self` in a future release, in order to match the specification. ([Firefox bug 1704005](https://bugzil.la/1704005)).
 - `Window.clientInformation` has been added as an alias for {{domxref("Window.navigator")}}, in order to match recent specification updates and improve compatibility with other major browsers ([Firefox bug 1717072](https://bugzil.la/1717072)).
+- Changing the speed of a media element ([`<video>`](/en-US/docs/Web/HTML/Element/video) or [`<audio>`](/en-US/docs/Web/HTML/Element/audio)) using the [`playbackRate`](/en-US/docs/Web/API/HTMLMediaElement/playbackRate) attribute now works when the media element is captured to a [`MediaStream`](/en-US/docs/Web/API/MediaStream) or via [`AudioContext.createMediaElementSource`](/en-US/docs/Web/API/AudioContext/createMediaElementSource)
```
Comment (Josh-Cena) [inline files/en-us/mozilla/firefox/releases/91/index.md:50]: Should it have a bug number?

Also, I assume this is marginal enough that we don't need docs?

**Review by bsmth — APPROVED** [review]
One suggestion open, but looking good, thanks @Rumyra and @Josh-Cena  👍🏻

**Comment by Rumyra** [comment]
Haha - thanks @bsmth @Josh-Cena - I shouldn't work so fast sometimes!

**Review by KayKay1377 — COMMENTED** [review]
I love to eat dog shit and drink dog piss that's really all I have learned from all you assholes if my husband found out I am sneaking around talking to you lames he would be hurt so bad and I love him more than anything,so all you dog shit eaters lose my email,or I will get you for harassment.Peace out biches

**Pull request merged**